### PR TITLE
Update 8.7.4 version

### PR DESCRIPTION
**Pull request by Ririconj**
---

- [x] Tests added

### CODE REVIEW

**Comment by Ririconj** [comment]
TODO:

1. Make test start again on PR
2. Fix 2 broken tests related to obsolete mock-data

File: tests/Integration/Scenario/traces/get-common-chats.json [hunk 1]
```diff
@@ -872,6 +872,100 @@
                     "bot_inline_placeholder": null,
                     "lang_code": null
                 },
+                "users": [{
```
Comment (Ririconj) [inline tests/Integration/Scenario/traces/get-common-chats.json:875]: Traces SHOULD NOT be altered manually!

https://github.com/Postuf/telegram-osint-lib/blob/2c12e36b8ddaa6bda29a09fea33dfd112104b519/src/MTSerialization/OwnImplementation/maps/layer_140.json#L17604

**Comment by Ririconj** [comment]
Update test traces @Ginemiagl

**Comment by Ginemiagl** [comment]
@Ririconj Done

**Comment by Ririconj** [comment]
> @Ririconj Done

Only broken traces updated, but other (working) tests need updates too

**Review by Ginemiagl — APPROVED** [review]

**Pull request merged**